Tests/XCUITests/DatabaseFixtureTest.swift

```diff
@@ -32,29 +44,179 @@ class DatabaseFixtureTest: BaseTestCase {
+//    }
+
+    func testPerfHistory1startUp() {
         waitForTabsButton()
-        navigator.goto(LibraryPanel_Bookmarks)
-        waitForExistence(app.tables["Bookmarks List"], timeout: TIMEOUT_LONG)
+        measure(metrics: [
+            XCTMemoryMetric(),
+            XCTClockMetric(), // to measure timeClock Mon
+            XCTCPUMetric(), // to measure cpu cycles
+            XCTStorageMetric(), // to measure storage consuming
+            XCTMemoryMetric()]) {
+                // activity measurement here
+                app.launch()
+                waitForTabsButton()
+        }
+    }
+
+    func testPerfHistory1openMenu() {
+        waitForTabsButton()
+        measure(metrics: [
+            XCTMemoryMetric(),
+            XCTClockMetric(), // to measure timeClock Mon
+            XCTCPUMetric(), // to measure cpu cycles
+            XCTStorageMetric(), // to measure storage consuming
+            XCTMemoryMetric()]) {
+                navigator.goto(LibraryPanel_History)
+                let historyList = app.tables["History List"]
+                waitForExistence(historyList, timeout: TIMEOUT_LONG)
+                let expectedCount = 2
+                XCTAssertEqual(historyList.cells.count, expectedCount, "Number of cells in 'History List' is not equal to \(expectedCount)")
+        }
+    }
 
-        let loaded = NSPredicate(format: "count == 1001")
-        expectation(for: loaded, evaluatedWith: app.tables["Bookmarks List"].cells, handler: nil)
-        waitForExpectations(timeout: TIMEOUT_LONG, handler: nil)
+    func testPerfHistory100startUp() {
+        waitForTabsButton()
+        measure(metrics: [
+            XCTMemoryMetric(),
+            XCTClockMetric(), // to measure timeClock Mon
+            XCTCPUMetric(), // to measure cpu cycles
+            XCTStorageMetric(), // to measure storage consuming
+            XCTMemoryMetric()]) {
+                // activity measurement here
+                app.launch()
+                waitForTabsButton()
+        }
+    }
+
+    func testPerfHistory100openMenu() {
+        waitForTabsButton()
+        measure(metrics: [
+            XCTMemoryMetric(),
+            XCTClockMetric(), // to measure timeClock Mon
+            XCTCPUMetric(), // to measure cpu cycles
+            XCTStorageMetric(), // to measure storage consuming
+            XCTMemoryMetric()]) {
+                navigator.goto(LibraryPanel_History)
+                let historyList = app.tables["History List"]
+                waitForExistence(historyList, timeout: TIMEOUT_LONG)
+                let expectedCount = 101
+                XCTAssertEqual(historyList.cells.count, expectedCount, "Number of cells in 'History List' is not equal to \(expectedCount)")
+        }
+    }
 
-        let bookmarksList = app.tables["Bookmarks List"].cells.count
-        XCTAssertEqual(bookmarksList, 1001, "There should be an entry in the bookmarks list")
+    func testPerfHistory200startUp() {
```

Comment:
I guess you know but just to note here: same as above, not a db with 200 entries